<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-TLOZ" ignored="0" name="Test" tests="3" failures="1" timestamp="2020-05-07T13:12:14 MDT" time="25.37" errors="0">
  <testcase name="h" time="0.009" classname="com.regression.Flight_Test_Singleton">
    <skipped/>
  </testcase> <!-- h -->
  <testcase name="f" time="6.694" classname="com.regression.Test_1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.regression.Test_1.f(Test_1.java:21)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="23.931" classname="com.regression.Test_2"/>
</testsuite> <!-- Test -->
